spec/lib/whitehall_importer/import_spec.rb

```diff
@@ -59,5 +58,35 @@
 
       expect(document.created_by).to eq(user)
     end
+
+    it "sets current boolean on whether edition is current or not" do
+      past_edition = build(
+        :whitehall_export_edition,
+        created_at: Time.zone.now.yesterday.rfc3339,
+        revision_history: [
+          { "event" => "create", "state" => "draft", "whodunnit" => whitehall_user["id"] },
+        ],
+      )
+      current_edition = build(
+        :whitehall_export_edition,
+        revision_history: [
+          { "event" => "create", "state" => "draft", "whodunnit" => whitehall_user["id"] },
+        ],
+      )
+
+      import_data = build(:whitehall_export_document,
+                          editions: [past_edition, current_edition],
+                          users: [whitehall_user])
+
+      expect(WhitehallImporter::CreateEdition).to receive(:call).with(
+        hash_including(current: false),
+      ).ordered
+
+      expect(WhitehallImporter::CreateEdition).to receive(:call).with(
+        hash_including(current: true),
+      ).ordered
+
+      described_class.call(import_data)
```

Comment:
If we set up a revision history factory can this be simpler at all?

Reply:
it certainly can!